File: app/src/main/java/io/zenandroid/greenfield/model/Image.java
package io.zenandroid.greenfield.model;

import java.util.Date;

/**
 * Created by alex on 24/01/2018.
 */

public class Image {
    private String title;
    private String link;
    private MediaLink media;
    private Date dateTaken;
    private String description;
    private Date published;
    private String author;
    private String authorId;
    private String tags;

    public MediaLink getMedia() {
        return media;
    }

    public String getTitle() {
        return title;
    }

    public String getAuthor() {
        return author;
    }

    public Date getPublishedDate() {
        return published;
    }

    public String getDescription() {
        return description;
    }

    public String getTags() {
        return tags;
    }

    public Date getDateTaken() {
        return dateTaken;
    }

    public String getLink() {
        return link;
    }

}

File: app/src/main/java/io/zenandroid/greenfield/model/Image.kt
package io.zenandroid.greenfield.model

import java.util.*

/**
 * Created by alex on 24/01/2018.
 */

data class Image (
    val title: String? = null,
    val link: String? = null,
    val media: MediaLink? = null,
    val dateTaken: Date? = null,
    val description: String? = null,
    val published: Date? = null,
    val author: String? = null,
    val authorId: String? = null,
    val tags: String? = null
)
